coverage_assessment.R

```diff
@@ -42,6 +42,140 @@ nut <- get_data()
 # Problem 3: Prevalence of acute undernutrition/malnutrition ----
 
 
+SEX <- ifelse(nut$sex == 1, "Male", "Female") ##Assigning male and female to sex
+
+OEDEMA <- ifelse(nut$oedema == 1, "Yes", "No") ## Assigning yes and No to oedema
+
+COUNTY <- nut$county ##Assigning nut$county to COUNTY
+
+Den <- sum(nut$age >=6 & nut$age<60, na.rm = TRUE) ## Assigning the total number of children between 6-60 to Den
+
+
+####NUMBER AND PREVALENCE OF MAM CASES
+
+
+Mod_MUAC <- with (nut, muac>= 11.5 & muac <12.5 & age >=6 & age<60) ## Assigning the moderate MUAC criteria for children between 6-60 for Mod_MUAC
```

Comment:
Moderate wasting should also indicate that there is no oedema. Any nutrition oedema case is automatically severe even if MUAC is 11.5 or greater.